src/typeahead/typeahead.directive.ts

```diff
@@ -159,13 +162,6 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
       e.preventDefault();
       return;
     }
-
-    // if tab default browser behavior will select next input field, and
-    // therefore we should close the items list
-    if (e.keyCode === 9) {
```

Comment:
all looks good except of this change, it should not be a part of adding onBlur output